src/components/submission/show.vue

```diff
@@ -100,14 +99,23 @@ export default {
       ? [`${t('title.details')}: ${submission.instanceNameOrId}`]
       : [t('title.details')]));
 
-    const { formPath } = useRoutes();
+    const { formPath, projectPath } = useRoutes();
     return {
-      project, submission, submissionVersion, audits, comments, diffs, fields,
+      project, form, submission, submissionVersion, audits, comments, diffs, fields,
       request, awaitingResponse, ...resourceStates([project, submission]),
```

Comment:
add `form` to resourceStates here so it can be used by dataExists and initiallyLoading